plugins/woocommerce/src/Admin/Notes/NoteTraits.php

```diff
@@ -164,13 +164,21 @@ public static function possibly_update_note() {
 			return;
 		}
 
-		// Update note content if it's changed.
-		$latest_note_content = $note->get_content();
-		if ( $note_in_db->get_content() !== $latest_note_content ) {
-			$note_in_db->set_content( $latest_note_content );
+		$need_save = false;
```

Comment:
I know nothing about PHP but I wonder if this would be cleaner? https://stackoverflow.com/questions/6850452/check-if-multiple-values-are-all-false-or-all-true/6850548#6850548

Reply:
This looks nice! I also know little about PHP. 😅

Reply:
Updated in 5b294e4c11bf67b2435761bbe2a76f2b2cbdf007.